### ISSUE TITLE
python documentation

### ISSUE THREAD

**Issue by paddyroddy** [astro-informatics/s2let#18]
Ideally something like https://github.com/astro-informatics/ssht/blob/public/src/python/SSHT_Python_Documentation.md

**Comment by mousset** [comment]
Yes, It would be great to have a python documentation as the one given in the example above or directly in the function docstrings.

**Comment by PhyAMR** [comment]
Is there still no python documentation? I can not find it and guessing what each function does based on the description of it is almost imposible. Also, is there a way to define custom wavelets?

**Comment by jasonmcewen** [comment]
Note that `s2let` has been superseeded by [`s2wav`](https://github.com/astro-informatics/s2wav), which implements the same underlying scale discrised wavelets on the sphere but natively in Python with JAX, thus providing support for automatic differentiation and GPU acceleration.  `s2wav` is also reasonably well documented (see [documentation here](https://astro-informatics.github.io/s2wav/)).  Due to the new code we won't be extending `s2let` any further.  Hopefully `s2wav` should meet your needs!
